### DISCUSSION TITLE
Collaborative editing with yjs

### DISCUSSION THREAD

**Discussion by jakedahn** [facebook/lexical#2581] [Q&A]
I'm interested in trying out Lexical + Yjs for CRDT collaborative editing, but I've been running into issues with the minimal  collaboration example that I found here: https://lexical.dev/docs/collaboration/react

I'm seeing the error `ReferenceError: WebSocket is not defined` which is coming from the `y-websocket` package.

Before I dive too far into the weeds, I was curious if there's any sort of updated documentation or examples for collaborative editing with lexical + yjs?

**Comment by Bunkerbewohner** [comment]
A bit late, but since this is still quite far at the top, maybe your code was executed on the server (Node.js) where `WebSocket` is not defined by default? See https://github.com/yjs/y-websocket#client-code-in-nodejs

**Comment by SalahAdDin** [comment]
[Here](https://liveblocks.io/docs/guides/how-to-create-a-collaborative-text-editor-with-lexical-yjs-nextjs-and-liveblocks) a tutorial with other technologies.
